internal/worker/worker.go

```diff
@@ -196,6 +206,51 @@ func (worker *Worker) Run(ctx context.Context) error {
 	}
 }
 
+func (worker *Worker) tryCreateStandby(ctx context.Context) {
+	// Do nothing if no standby instance is configured
+	if worker.standbyIsolation == nil {
+		return
+	}
+
+	// Do nothing if the standby instance is already instantiated
+	if worker.standbyInstance != nil {
+		return
+	}
+
+	// Do nothing if there are tasks that are running to simplify the resource management
+	if worker.tasks.Size() != 0 {
```

Comment:
I think it's OK for now but we'll need to think how to make a standby if we run one VM already.